process/elasticproc/converters/field.go

```diff
@@ -0,0 +1,22 @@
+package converters
+
+import "github.com/multiversx/mx-chain-es-indexer-go/data"
+
+// TruncateFieldIfExceedsMaxLength will truncate the provided field if the max length exceeds
+func TruncateFieldIfExceedsMaxLength(field string) string {
+	if len(field) > data.MaxFieldLength {
+		return field[:data.MaxFieldLength]
+	}
+
+	return field
+}
+
+// TruncateSliceElementsIfExceedsMaxLength will truncate the provided slice of field if the max length exceeds
```

Comment:
// TruncateSliceElementsIfExceedsMaxLength will truncate the provided slice of the field if the max length is exceeded

Reply:
done